### PR TITLE
Added theme support to a11y mode

### PR DESCRIPTION
**Pull request by zainafzal08**
Has the a11y area correctly update with theme change

### CODE REVIEW

File: css/app.css [hunk 1]
```diff
@@ -525,7 +539,18 @@ header.hide-controls #window-close {
   overflow: hidden;
 }
 
-#editor-textarea:focus {
+#editor-textarea:focus, #editor-textarea:hover {
+  outline: none;
+}
+
```
Comment (BugsNash) [inline css/app.css]: nit: remove second blank line

File: css/app.css [hunk 2]
```diff
@@ -525,7 +539,18 @@ header.hide-controls #window-close {
...
+
+.editor-textarea-container.dark-theme .editor-textarea-wrapper {
+  background: var(--dark-background-color);
+  color: var(--dark-theme-text-color);
+}
+
+#ed
```
Comment (BugsNash) [inline css/app.css]: i'm guessing this newline is a mistake

File: js/editor-ta.js [hunk 3]
```diff
@@ -327,8 +326,13 @@ EditorTextArea.prototype.setTabSize = function(size) {
 /**
  * @param {string} theme
  */
-EditorTextArea.prototype.setTheme = function(theme) {
-  // TODO(zafzal): this.
+EditorTextArea.prototype.setTheme = function() {
+
```
Comment (BugsNash) [inline js/editor-ta.js]: nit: remove blank line

File: css/app.css (hunk 2 above)
Comment (zainafzal08) [reply css/app.css]: yep haha, interesting that it didn't effect behavior

**Comment by zainafzal08** [comment]
image
image
image

**Pull request merged**